ci-operator/step-registry/cucushift/installer/check/aws/private/cucushift-installer-check-aws-private-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: cucushift-installer-check-aws-private
```

Comment:
is the name `cucushift-installer-check-aws-private-route53` more suitable for this step?

Reply:
>   documentation: >-
>    Verify AWS private cluster
Oh, I just saw the doc description, then the current name is good for me.

Reply:
do we need any other checks against cluster itself?

Reply:
it's verify interesting that in the first I was assembling the whole provision chain as 
```
  - ...
  - ref: enable-qe-catalogsource
  - ref: proxy-config-generate
  - ref: aws-provision-public-to-private
  - chain: cucushift-installer-check
```
but in `chain: cucushift-installer-check` we have a [CPMS check](https://steps.ci.openshift.org/reference/cucushift-installer-check-control-plane-machinesets#line134) that's aimed for making sure there's no master created with name like master-xxxx-[0,1,2], but in this case, all the three masters will be re-created and named with master-xxxx-[0,1,2], so I added a new step `cucushift-installer-check-aws-private` here to check the **private** things for the cluster specifically, and I believe it could be used for the common AWS private cluster post checks later.

For the other cluster checks, I think the `cucushift-installer-check-cluster-health ` step
in the job definition https://github.com/openshift/release/pull/59213/files#diff-e6a92ef73c903cce61b8139c9cdff799e0766d7c933c183b0e51b6d681d2c1f4R192 should be enough for it.

Reply:
the behavior of master recreated is because we changed machine-api config on https://github.com/openshift/release/pull/59213/files#diff-fa0159cfe095118e1ced30d53c6eea8cd6e31ad49f44cc62fbd7d281033c0c00R95